lib/till.rb

```diff
@@ -0,0 +1,40 @@
+require 'json'
+require_relative 'order'
+
+TAX = 8.64
+
+class Till
+  attr_reader :shopName, :address, :phone, :prices, :orders, :currentOrder
+
+  def initialize
+    file = File.read('hipstercoffee.json')
+    data = JSON.parse(file)
+    @shopName = data[0]['shopName']
+    @address = data[0]['address']
+    @phone = data[0]['phone']
+    @prices = data[0]['prices'][0]
+    @orders = []
+    @currentOrder = []
+    self.newOrder
+  end
+
+  def printPrices
+    self.prices.each {|k,v| puts "#{k}: #{v}"}
```

Comment:
Space missing after comma.<br>Space between { and | missing.<br>Space missing inside }.<br>Do not write to stdout. Use Rails' logger if you want to log.